src/common/components/Navbar.vue

```diff
@@ -391,6 +391,11 @@ export default {
     },
 
     signOut () {
+      const accounts = Object.keys(window.localStorage).filter((v) =>
+        /account:/.test(v)
+      )
+      window.localStorage.removeItem(accounts[0])
```

Comment:
Can we compare it with the current account: instead?

Reply:
kindly re-review @bukandicki